tests/task_comments/test_api_modify_task_comment.py

```diff
@@ -88,7 +88,7 @@ def test_task_comment_modification_api_not_created_by_user(self):
             data=json.dumps(dict(comment="comment")),
         )
 
-        self.assertEqual(400, actual_response.status_code)
+        self.assertEqual(403, actual_response.status_code)
```

Comment:
@gaurivn use `HTTPStatus.FORBIDDEN.value` instead

Reply:
@devkapilbansal I think this can be ignored since a separate issue #968 is created for the same and the PR for that is yet to be created . So if this is merged tested and merged before PR for #968 is created it won't be a problem .
What say?  @isabelcosta @devkapilbansal

Reply:
Okay @vj-codes , then it is fine :+1: